src/app/Routing.tsx

```diff
@@ -6,6 +6,7 @@ import CreateBusinessplan from "./businessplan/create-businessplan/CreateBusines
 import LoginPage from "./login/LoginPage.tsx";
 import LoginFormPage from "./login/loginForm/LoginFormPage.tsx";
 import NoMatch from "./no-match/NoMatch";
+import ProfilPage from "./profil/ProfilPage.tsx";
```

Comment:
Although profilePage has been renamed it still appears to me as ProfilPage, weird, linter shouldn't let it pass -. -

Reply:
![obraz](https://github.com/szymonkadas/Lesson4Future2.0-Setup/assets/133753432/e66897b9-6f41-49d2-844e-a81c546341ed)